certbot-nginx/src/certbot_nginx/_internal/configurator.py

```diff
@@ -149,15 +149,24 @@ def mod_ssl_conf_src(self) -> str:
         # many versions of OpenSSL. So we have to check both for the two different features,
         # leading to four different combinations of options.
         # For a complete history, check out https://github.com/certbot/certbot/issues/7322
+        #
+        # Technically, nginx >= 1.23.2 has session tickets off by default, and therefore
+        # no longer needs it explicitly set. But since older versions then that are still
```

Comment:
```suggestion
        # no longer needs it explicitly set. But since older versions than that are still
```

Reply:
oh noooo I copied this from the apache comment. ah well